components/brave_ads/browser/notification_helper_win.cc

```diff
@@ -27,4 +113,129 @@ NotificationHelper* NotificationHelper::GetInstance() {
   return NotificationHelperWin::GetInstance();
 }
 
+///////////////////////////////////////////////////////////////////////////////
+
+bool NotificationHelperWin::IsFocusAssistEnabled() const {
+  const auto nt_query_wnf_stated_data_func =
```

Comment:
doesn't this need a null check?

Reply:
@bridiver This does not require a null check as the next call fails if this is null